msrc-appconfig-attrs/tests/appconfig_attrs_test.py

```diff
@@ -40,7 +40,7 @@ class Base:
 
 @attr.s()
 class NoTypeAnnotation():
-    f = attr.ib()
+    f = attr.ib()  # type: ignore
```

Comment:
CI was failing because pyright was failing on this line. I think Pyright has added attrs support since the last CI run